templates/ai-ml.html

```diff
@@ -1,374 +1,419 @@
+        <h2>NVIDIAとCanonicalがあらゆる環境でAIを加速します</h2>
+      </div>
+      <div class="col">
+        <p>NVIDIAとCanonicalは、あらゆるパブリッククラウド、オンプレミス、IoT機器でAIハードウェアアクセラレーションを提供できるよう協力しています。</p>
+        <hr class="is-muted" />
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">機械学習向けに最適化されたNVIDIA DGXサーバーはUbuntuを搭載しており、Canonicalによるサポートが含まれています</li>
+          <li class="p-list__item is-ticked">
+            ハードウェアアクセラレーションのためのNVIDIA Kubernetesエクステンションは、CanonicalのKubernetesディストリビューションで有効化されています
+          </li>
+          <li class="p-list__item is-ticked">NVIDIA TegraとDrivePX2にはエッジAI向けUbuntuが搭載されています</li>
+        </ul>
+      </div>
     </div>
-  </div>
-</section>
-<section class="p-strip is-bordered" id="install-kubeflow">
-  <div class="row">
-    <div class="col-12">
-      <h2>Kubeflowを自分でインストールする</h2>
-      <p>Canonicalは開発者や運用者を支援するために継続的なベストプラクティスの革新と開発を行っています。</p>
-      <h3>KubeflowをKubernetesにデプロイ</h3>
-      <ol class="p-stepped-list--detailed">
-        <li class="p-stepped-list__item">
-          <h3 class="p-stepped-list__title">Kubernetesのインストール</h3>
-          <div class="p-stepped-list__content">
-            <p>KubeflowはKubernetes上で動作します。Canonical KubernetesがKubeflowの最適な基盤です。</p>
-            <p>Kubernetesを独自にインストールするには、<a href="https://ubuntu.com/kubernetes/install" class="p-link--external">Kubernetesのインストール</a>のページにアクセスし、インストールガイドに従ってください。</p>
-          </div>
-        </li>
-        <li class="p-stepped-list__item">
-          <h3 class="p-stepped-list__title">MicroK8sのインストール</h3>
-          <div class="p-stepped-list__content">
-            <p>MicroK8sをインストールするbashスクリプトを入手します</p>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>wget https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/b783a95d-install-kubeflow-pre-micro.sh -O install-kubeflow-pre-micro.sh</code></pre>
-            </div>
-            <p>スクリプトを実行できるようにパーミッションを設定します</p>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>chmod a+x install-kubeflow-pre-micro.sh</code></pre>
-            </div>
-            <p>スクリプトをルートとして実行します</p>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>sudo ./install-kubeflow-pre-micro.sh</code></pre>
-            </div>
-          </div>
-        </li>
-        <li class="p-stepped-list__item">
-          <h3 class="p-stepped-list__title">環境変数の設定</h3>
-          <div class="p-stepped-list__content">
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>export KUBECONFIG=/snap/microk8s/current/client.config</code></pre>
-            </div>
-            <div class="p-code-snippet">
-              <pre class="p-code-snippet__block--icon"><code>export GITHUB_TOKEN=${YOUR_GITHUB_TOKEN}</code></pre>
-            </div>
+  </section>
+
+  <section class="p-section">
```

Comment:
According to the doc there's a missing link in this section, but I'm not sure if that's out of date. I left a [comment](https://docs.google.com/document/d/186nJQj0P5NTZR3CKnPkjq5I8kupCy1boHnOoHQZAHNI/edit?disco=AAABUUW4ayA)